package.json

```diff
@@ -1,13 +1,14 @@
 {
   "name": "ocs-component-lib",
-  "version": "0.12.4",
+  "version": "0.12.5",
   "private": false,
   "description": "Component library for an astronomical observatory control system frontend",
   "scripts": {
     "build": "vue-cli-service build --target lib --name ocs-component-lib ./src/main.js",
     "test:unit": "vue-cli-service test:unit",
     "lint:check": "vue-cli-service lint --no-fix --max-errors 0 --max-warnings 0",
-    "lint:fix": "vue-cli-service lint"
+    "lint:fix": "vue-cli-service lint",
+    "prepare": "npm run build"
```

Comment:
The "prepare" script is run automatically when installing from git, so this allows you to potentially debug by installing this npm package directly from a github branch or tag

Reply:
That's cool!